changelog/unreleased/issue-3464

```diff
@@ -1,9 +1,10 @@
-Enhancement: Skip lock creation on forget if --no-lock and --dry-run
+Enhancement: Skip lock creation on `forget` if `--no-lock` and `--dry-run`
 
-Restic used to silently ignore --no-lock option of forget. It now skips
-creation of lock file in case of both --dry-run and --no-lock are specified. If
---no-lock option is specified without --dry-run then restic prints a warning
-message to stderr.
+Restic used to silently ignore the `--no-lock` option of the `forget` command.
+
+It now skips creation of lock file in case of both `--dry-run` and `--no-lock`
+being specified. If `--no-lock` option is specified without `--dry-run`, then
```

Comment:
```suggestion
It now skips creation of lock file in case both `--dry-run` and `--no-lock`
are specified. If `--no-lock` option is specified without `--dry-run`,
```